lib/gds_api/performance_platform/data_out.rb

```diff
@@ -43,6 +43,119 @@ class DataOut < GdsApi::Base
       def service_feedback(transaction_page_slug)
         get_json("#{endpoint}/data/#{transaction_page_slug}/customer-satisfaction")
       end
+
+      # Fetches data from the performance platform for a given page slug
+      #
+      # Makes a +GET+ request.
+      #
+      # @param slug [String] The slug for which page is needed.
+      # @param group_by [String] The group by condition by which
+      # to group the results
+      # @param collect [String] The operation that should be applied
+      # on the results after they've been grouped
+      #
+      # # @example
+      #
+      #  performance_platform_data_out.search_terms('/european-health-insurance-card', 'pagePath', 'uniquePageviews:sum')
+
+      def page_statistics(slug, group_by, collect, is_multipart)
+        params = {
+            group_by: group_by,
+            collect: collect,
+            duration: 42,
+            period: "day",
+            end_at: Date.new.to_time.getutc
```

Comment:
This should be `Date.today` rather than `Date.new` as Date.new returns something distant like `"-4712-01-01"` (may be system-dependant).  We may have to worry about timezones, but I don't think we can rely on activesupport so we can't do `Time.zone.today` instead which is a shame.

We'll also want to add `.iso8601` to the end to turn the Time object into a good representation for sending across the wire.  `query_string` simply calls `to_s` on each param value and `to_s` on a Time instance gives us something like: `"2017-06-20 23:00:00 UTC"`, if we call `iso8601` first we get `"2017-06-20T23:00:00Z"`.  It's not clear to me if backdrop will properly parse the former and give us the correct results, but the example urls I saw in kibana were all using the latter so it's better to be safe.

This comment applies to all the `Date.new` bits.